### PR TITLE
data: analizzi

### PR DESCRIPTION
**Pull request by analizzi**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5317"></a>

### CODE REVIEW

**Review by Panquesito7 — DISMISSED** [review]
Thanks! 🎉

**Review by SurajPratap10 — DISMISSED** [review]

File: data/analizzi.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Ana Lizzi",
+  "type": "personal",
+  "bio": "Civil engineer by day, **coder** by night. Travel enthusiast Yoga & Peloton aficionado"
```
Comment (amandamartin-dev) [inline data/analizzi.json:4]:
```suggestion
  "bio": "Civil engineer by day, **coder** by night. Travel enthusiast Yoga & Peloton aficionado",
```

**Review by amandamartin-dev — CHANGES_REQUESTED** [review]
missing a comma

**Comment by eddiejaoude** [comment]
Duplicate of https://github.com/EddieHubCommunity/LinkFree/pull/5319

**Review by eddiejaoude — APPROVED** [review]
Looks great 👍 

Great collaboration everyone 🎉

**Pull request merged**